### DISCUSSION TITLE
NvimTreeFocus for <leader>e . A better default ?

### DISCUSSION THREAD

**Discussion by cassepipe** [LunarVim/LunarVim#4084] [Ideas]
I have made the following remapping in my config :
```lua
lvim.builtin.which_key.mappings["e"] = { ":NvimTreeFocus<CR>", "Explorer" }
``` 
The idea is that `<leader>e` will focus the explorer instead of closing it when it's already opened.

Why I think that 's better :
- It happens more than you want to refocus the explorer than you want to close it. Focusing a window is not straightforward since it's a sequence with a combination ( `<C-w>h`) (if your explorer is on the left) vs. `<leader>e` that is very quick.
- It's super easy to close the explorer when it's focused, just type `q`. If the explorer is not focused --> `<leader>eq`

What do you think ?

**Comment by brneor** [comment]
I like `<leader> e` for toggling NvimTree as I prefer a quick way to close it and give me back precious screen space. Otherwise I'd have to focus it, close it and then move back to the buffer I was on.

**Reply by cassepipe** [reply]
My screen space is just as precious as yours on a 13' laptop :)
The thing is you don't need to move back to the buffer you were on, that's where you end up when you close the explorer.

Focus and close is just would be just that sequence: `<leader>eq`

I guess you don't have that problem when you want to open various files from the explorer ? If you do how do you refocus the explorer without closing it (since opening a file will focus it ?)

**Reply by brneor** [reply]
I actually use `<leader> f` to open files most of the time 😅
